lib/runner/TestRunner.js

```diff
@@ -67,6 +71,10 @@ module.exports = class TestRunner {
             }
 
             testResult.interactionResults = results;
+            
+            testSerialized.interactionResults = results.map(result => result.toString());
```

Comment:
Why does it need to be serialized?

Reply:
the object have circular structures, it was easier to debug with a serialized version. But I can return the whole object.

Reply:
Okay, I understand - I would not call it "toString" then - that implies something different in my mind - how about serialize() or toDTO()? Read here for an explanation of DTO if you are not familiar with it:
https://en.wikipedia.org/wiki/Data_transfer_object